### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by useroqg9g** [Daksh777/SpotifyNoPremium#21] [Q&A] [answered]
I've already uninstalled Git, Spicetify and Blockthespot. How do I delete the remaining files, is there anything else to delete?

**Comment by Daksh777** [comment, marked as answer]
First, you had to restore your Spotify version to the stock state by using `spicetify restore` but since you have already uninstalled Spicetify, you can just reinstall Spotify.
Other than that, you have already removed what was needed

**Reply by c0d3xrz** [reply]
Thanks, it helped a lot👍
